_includes/cockroachcloud/ccloud/backup-types.md

```diff
@@ -0,0 +1,4 @@
+{{ site.data.products.db }} clusters support two types of backups:
+
+- Managed-service backups: Cockroach Labs takes automated backups of **{{ site.data.products.serverless }}** and **{{ site.data.products.dedicated }}** clusters that are stored in Cockroach Labs' cloud storage. See the [Restore Data from a Backup](../cockroachcloud/backups-page.html) page for the type and frequency of backups for both {{ site.data.products.db }} clusters.
```

Comment:
```suggestion
- **Managed-service backups**: Cockroach Labs takes automated backups of {{ site.data.products.serverless }} and {{ site.data.products.dedicated }} clusters that are stored in Cockroach Labs' cloud storage. See the [Restore Data from a Backup](../cockroachcloud/backups-page.html) page for the type and frequency of backups for both {{ site.data.products.db }} clusters.
```

Reply:
Sorry if this wasn't clear, I intended for the names of the backup types to be in bold rather than the product names here.